Cloud Application Manager/Automating Deployments/version-control.md

```diff
@@ -1,82 +1,102 @@
 
 From here, you can update the instance with a specific version. An update always triggers a [reinstall operation](../Deploying Anywhere/deploying-managing-instances.md).
 
 ![version-control-and-auto-updates-7.png](../../images/cloud-application-manager/version-control-and-auto-updates-7.png)
 
-**Note:** When deploying a box from the Instances page, you consume the latest configuration of the parent and child boxes unless you chose a specific version for the child boxes. To deploy a different version of the parent box, launch it from the box page. From the Versions tab > gear icon, click **Deploy** on a select version.
+**Note:** When deploying a box from Instances page, you consume the latest configuration of the parent and child boxes unless you chose a specific version for the child boxes. To deploy a different version of the parent box, launch it from the box page. From the **Versions** tab > gear icon, click **Deploy** on a selected version.
 
-### Manage a version box
+### Managing box versions
 
-Version boxes can be managed in different ways. To see these options, go to versions tab and select gear icon.
+Box versions can be managed in different ways. To see these options, go to **Versions** tab and click the gear icon.
 
 ![version-control-and-auto-updates-8.png](../../images/cloud-application-manager/version-control-and-auto-updates-8.png)
 
-1. **Delete**: You can delete the selected version box. Please be aware that clicking on this option deletes the version without a warning popup.
-2. **Deploy**: You can deploy the selected version box.
-3. **Diff**: You can see the updates made to each version box from the versions tab. Select gear icon and click on **Diff**.
+1. **Delete**: You can delete the selected box version. Please be aware that clicking on this option deletes the version without a warning popup.
+2. **Deploy**: You can deploy the selected box version.
+3. **Diff**: You can see the updates made to each box version from the **Versions** tab. Select the gear icon and click on **Diff**.
 
 ![version-control-and-auto-updates-9.png](../../images/cloud-application-manager/version-control-and-auto-updates-9.png)
 
-4. **Restore**: You can restore your version box to its previous version. The version you are working on now will turn into the current version.
-5. **Publish box**: This option changes the privacy settings of the version box, making it public, after being published.  To execute this change, complete the request form. Please note that only those fields marked with an asterisk are mandatory.
+4. **Restore**: You can restore your box to its previous version. The version you are working on now will become the current version.
+5. **Publish box**: This option changes the privacy settings of the box, making it public after being published. To execute this change filling a request form is necessary. Please note that only those fields marked with an asterisk are mandatory.
```

Comment:
Please, link to an article explaining the publish box feature

Reply:
You already added it in a previous revision of the article. I'll move it before the picture to make it more visible if you agree.

Reply:
Yes, please :)